app/models/participant_profile.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class ParticipantProfile < ApplicationRecord
+  belongs_to :user
+  belongs_to :school
+  belongs_to :cohort
+  belongs_to :core_induction_programme, optional: true
+
+  def ect?
+    false
+  end
+
+  def mentor?
+    false
+  end
+
+  scope :mentors, -> { where(type: Mentor.name) }
+  scope :ects, -> { where(type: ECT.name) }
+
+  scope :sparsity, -> { where(sparsity_uplift: true) }
+  scope :pupil_premium, -> { where(pupil_premium_uplift: true) }
+  scope :uplift, -> { sparsity.or(pupil_premium) }
+
+  class ECT < self
+    belongs_to :mentor_profile, class_name: "Mentor", optional: true
+
+    def ect?
+      true
+    end
+  end
+
+  class Mentor < self
+    self.ignored_columns = %i[mentor_profile_id]
```

Comment:
What does this do?

Reply:
Not as much as I would like it to do - it removes the column from model inspection and from `attributes`. Annoyingly it does not remove getter and setter though...